### PR TITLE
:sparkles: add support for BillOfLadingV1, FR EnergyBillV1, NutritionFactsLabelV1, FR PayslipV1, UsMailV2

### PR DESCRIPTION
**Pull request by sebastianMindee**
## Description

* :sparkles: add support for Bill of lading V1
* :sparkles: add support for (FR) Energy Bill V1
* :sparkles: add support for Nutrition Facts Label V1
* :sparkles: add support for (FR) Payslips V1
* :sparkles: add support for (US) UsMailV2
* :recycle: tweak rst display to accommodate for new additions

## Types of changes

- [x] Docs change / refactoring / dependency upgrade
- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] Requires a change to the official Guide documentation.

### CODE REVIEW

File: mindee/parsing/standard/base.py [hunk 1]
```diff
@@ -167,7 +167,7 @@ def int_to_string(value: Optional[int]) -> str:
     if value is None:
         return ""
 
-    return f"{value}"
+    return f"{value}.00"
```
Comment (ianardee) [inline mindee/parsing/standard/base.py:170]: an int should never be rendered as a float

File: mindee/parsing/common/summary_helper.py [hunk 2]
```diff
@@ -22,6 +22,7 @@ def format_for_display(
     """Truncates line-items to the max width of their corresponding column."""
     if not out_string or len(out_string) == 0:
         return ""
+    out_string = out_string.replace("\n", "\\n")
```
Comment (ianardee) [inline mindee/parsing/common/summary_helper.py:25]: what about the other characters \r, \t

File: mindee/parsing/standard/base.py (hunk 1 above)
Comment (sebastianMindee) [reply mindee/parsing/standard/base.py:170]: Wow, I remember doing that, but it feels like a fever dream now :woozy_face:

File: mindee/parsing/common/summary_helper.py (hunk 2 above)
Comment (sebastianMindee) [reply mindee/parsing/common/summary_helper.py:25]: Forgot to push latest changes, my bad

**Review by ianardee — APPROVED** [review]

**Pull request merged**